cmake/GzSetCompilerFlags.cmake

```diff
@@ -347,14 +348,24 @@ macro(_gz_setup_msvc)
   # framework.
   #
   # In some cases, a user might want to compile with the static runtime. This
-  # should ONLY be done if they do not intend to use the ignition library as
+  # should ONLY be done if they do not intend to use the Gazebo library as
   # part of a plugin-based framework.
-  option(IGN_USE_STATIC_RUNTIME "Use the static runtime (strongly discouraged)" OFF)
+
+  if(IGN_USE_STATIC_RUNTIME)  # TODO(CH3): Deprecated. Remove on tock.
```

Comment:
Do we still need the `option()` line before checking if it's set?

Reply:
Option will create a cache variable which can then be set via CLI, if I'm not wrong